### ISSUE TITLE
Add webhook interface and export it

### ISSUE THREAD

**Issue by sachaw** [TypeCtrl/plex#13]
A number of my projects use plex webhooks, and every time I implement this functionality I have to create a new type definition for the expected structure, and it it ever changes, I then have to go and update all of my projects.
I think it would be a good addition to the library, just have it as one of the root level exports, so it can be updated in one place.

Here's what I have atm, could definitely be improved and some parts may be deprecated, or need updating, but it's a start:
https://github.com/sachaw/scrobble-moe/blob/main/utils/webhookInterface.ts

**Comment by scttcper** [comment]
So you want the interface for the plex request hitting you right? I think that makes sense, could be an exported type. I don't think there's a good way to write a function that takes a request and returns some sort of object is there.

**Comment by sachaw** [comment]
Yeah, as you said, I wouldn't bother writing any parser, just the type interface
